io-manager.js

```diff
@@ -195,10 +190,45 @@ const _updateIo = timeDiff => {
             !game.isJumping() &&
             !game.isSitting()
           ) {
-            game.jump();
+            game.jump('jump');
+          }
+
+          if (buttons[0] >= 0.5) {
+            game.menuMouseDown();
+          } else {
+            game.menuMouseUp();
+          }
+
+          if (buttons[1] >= 0.5) {
+            game.menuActivateDown();
           }
+
+          if (buttons[5] >= 0.5 && !ioManager.lastButtons[5]) {

```

Comment:
The buttons should not be queried directly like this. It should be mapped per controller.